lib/chef/knife/serve.rb

```diff
@@ -25,15 +26,15 @@ class Serve < Knife
 
       option :repo_mode,
         long: "--repo-mode MODE",
-        description: "Specifies the local repository layout. Values: static (only environments/roles/data_bags/cookbooks), everything (includes nodes/clients/users), hosted_everything (includes acls/groups/etc. for Enterprise/Hosted Chef). Default: everything/hosted_everything."
+        description: "Specifies the local repository layout. Values: static (only environments/roles/data_bags/cookbooks), everything (includes nodes/clients/users), hosted_everything (includes acls/groups/etc. for Enterprise/Hosted #{Chef::Dist::PRODUCT}). Default: everything/hosted_everything."
```

Comment:
I think this one needs to stay. Hosted Chef is a product and any client could be used with it. Making it Hosted Chef Infra Client doesn't make sense and making it Hosted Cinc Client certainly doesn't make sense.

Reply:
good catch, will fix